Source/WebCore/Modules/cookie-store/CookieStore.cpp

```diff
@@ -383,8 +384,10 @@ void CookieStore::set(CookieInit&& options, Ref<DeferredPromise>&& promise)
 
     // The maximum attribute value size is specified at https://wicg.github.io/cookie-store/#cookie-maximum-attribute-value-size.
     static constexpr auto maximumAttributeValueSize = 1024;
+
+    auto url = context->creationURL();
+    auto host = url.host().toString();
```

Comment:
Should just keep host as a StringView - We make a StringView out of it later, even!

Reply:
Good point--fixed!